### ISSUE TITLE
If you copy a minion with Rush from Goblin Lackey, the copied minion will not have Rush

### ISSUE THREAD

**Issue by troggnostupidhs** [HearthSim/hs-bugs#1146]
https://youtu.be/msHYVfnbpc8?t=197

https://www.reddit.com/r/hearthstone/comments/bcjctx/bug_with_rush_lackey_and_copying/

> I am pretty sure I just experienced a bug. My opponent had Hakkar and gave it Rush with the Goblin Lackey. I decided to faceless manipulator the Hakkar, in order to trade it into itself.
> 
> What happened instead was I got the buffed Hakkar but no Rush.
> 
> Clearly the minion should have Rush, no?
> 
> PSA - don't copy a Rushed minion from Goblin Lackey, you will be disappointed.
>

**Comment by Kybxd** [comment]
If you grant Immortal Prelate Rush with Goblin Lackey, trade Prelate, then draw and play it (or recruit it) in the same turn, it won't even keep the Rush.

**Comment by troggnostupidhs** [comment]
Jetz and I tested Boomship copying same side with same turn and next turn copying and the minion had Rush in all cases.

**Comment by Kybxd** [comment]
Yes, it seems that Rush granted from Goblin Lackey is somehow different from other Rush. Here is another bug of Goblin Lackey:

https://www.bilibili.com/video/av50517505 (3:13 to 4:25)

This video shows that if the minion who gets Rush from Goblin Lackey is silenced, it can still get +2/+1 from Woodcutter's Axe's deathrattle. This doesn't happen on Rabid Worgen or Rocket Boots.
